### PR TITLE
Bump and control the version of lodash in dev-dependencies

### PR DESCRIPTION
**Pull request by archmoj**
Addressing 12 audit warnings https://www.npmjs.com/advisories/1523.

@alexcjohnson

### CODE REVIEW

**Comment by archmoj** [comment]
Wondering if we should wait until https://github.com/lodash/lodash/issues/4844 is resolved?

File: package.json [hunk 1]
```diff
@@ -152,6 +152,7 @@
     "karma-spec-reporter": "0.0.32",
     "karma-verbose-reporter": "0.0.6",
     "karma-viewport": "^1.0.6",
+    "lodash": "^4.17.19",
```
Comment (alexcjohnson) [inline package.json:155]: do we need to add it here? Isn't it sufficient to have lodash bumped in `package-lock.json`?

**Comment by alexcjohnson** [comment]
> Wondering if we should wait until lodash/lodash#4844 is resolved?

Seems like if our builds and tests run, that issue isn't a problem for us, right? And anyway it's not clear yet that they're in any rush to fix it.

File: package.json (hunk 1 above)
Comment (archmoj) [reply package.json:155]: I think it is better to keep it this way e.g. in case one wants to regenerate the lock as well the for the yarn upgrade.

Comment (alexcjohnson) [reply package.json:155]: I just don't want us to be in the habit of including sub-dependencies in `package.json`, is this one special in some way? Are we waiting for something that will let us remove this?

Comment (archmoj) [reply package.json:155]: Until all those sub-dependencies switch to force using newer version, I think we need to control the version at root.
We have few other dev-dependencies e.g. `acorn`, `static-eval` that controlled like this.

Comment (alexcjohnson) [reply package.json:155]: OK, fair enough.

**Pull request merged**